icons/addons/services.svg

```diff
@@ -1,4 +1,4 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. --> 
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M0,4V8.7a10.45,10.45,0,0,0,0,1.84A3.1,3.1,0,0,0,3,13c1.28.17.86,1,1,2,.18,1.26,1.24,1.09,2,.51l2.34-1.75A2.65,2.65,0,0,1,9.77,13c1.76-.15,3.95.52,5.35-.88S16,8.63,16,7s.39-3.86-.88-5.12S12.36,1,10.92,1H4.38C3.92,1,3.46,1,3,1A3,3,0,0,0,0,4Zm14,6.2c0,1.38-3.81.78-4.67.8C8.05,11,7,12.26,6,13c0-.47.14-1.27-.16-1.67S4.7,11,4.2,11c-1,0-2.15.22-2.2-1.08S2,7.44,2,6.21A8.67,8.67,0,0,1,2,3.55C2.33,2.77,3.56,3,4.21,3h7.72c.64,0,1.74-.21,2,.55a3.47,3.47,0,0,1,0,1.11V10.2Z" fill="context-fill" /></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M4,15V14a1,1,0,0,0-1-1H3a3,3,0,0,1-3-3V4A3,3,0,0,1,3,1H13a3,3,0,0,1,3,3v6a3,3,0,0,1-3,3H10.34a3,3,0,0,0-1.8.6l-2.93,2.2A1,1,0,0,1,4,15Zm1-4a1,1,0,0,1,1,1v1l2.4-1.8A1,1,0,0,1,9,11h4a1,1,0,0,0,1-1V4a1,1,0,0,0-1-1H3A1,1,0,0,0,2,4v6a1,1,0,0,0,1,1Z" fill="context-fill"/></svg>
```

Comment:
Sometimes we indent and sometimes we do not. I would stick to only one convention :)